packages/cactus-cmd-socketio-server/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
@outSH Why do you need a separte jest config file?

Reply:
- cmd-socketio doesn't integrate well with the rest of the monorepo structure (yet), it uses local npm_modules (not top level modules hoisted by yarn) and different, not inherited tsconfig (with newer typescript version).
- Running jest from the root dir assumes global, common config which causes existing tests to fail (jest treats all the files as single project and fails due to some module conflicts).
- Fixing this is not a trivial task, we need to either add projects configuration in root jest config or do major adjustment in cmd-socketio itself, and I plan to investigate and fix this in another task/PR.
- For now, I've added separate jest config to allow running tests specific to cmd-socketio without breaking the CI.

Reply:
> cmd-socketio doesn't integrate well with the rest of the monorepo structure (yet), it uses local npm_modules (not top level modules hoisted by yarn) and different, not inherited tsconfig (with newer typescript version).

@outSH I sent a PR back in March that fixed all that, but then Takuma deleted the changes with a force push. Any chance that you could work off of what was in that PR?
https://github.com/hyperledger/cactus/pull/719

Reply:
> Any chance that you could work off of what was in that PR?
@petermetz Sure, I will look into that, but shall I include these changes in this or separate PR?

Reply:
@outSH Whichever is more efficient for you, I don't want to hold up progress, time is short. If that's better for you, we can merge this now as is if you send a link to an opened issue that deals with what I was talking about. Let me know!

Reply:
@petermetz I've created a separate issue for this problem here - #1647, please merge these tests if they look fine to you. I'll fix this in the near future or as part of #1502

Reply:
@outSH Thank you very much! GitHub only lets me assign the issue to you once you've commented on it (the contents can be anything, even a single character). Please header over to #1502 make a comment that you'd like to work on it (make sure to tag me so that it shows up on my GH feed) and then I'll assign it to you.